sdk/core/core-http/test/mockHttp.ts

```diff
@@ -32,31 +33,44 @@ export interface HttpMockFacade {
   get(url: UrlFilter, response: MockResponse): void;
   post(url: UrlFilter, response: MockResponse): void;
   put(url: UrlFilter, response: MockResponse): void;
+  getFetch(): typeof node_fetch | undefined;
 }
 
 export function getHttpMock(): HttpMockFacade {
   return isNode ? new FetchHttpMock() : new BrowserHttpMock();
 }
 
 class FetchHttpMock implements HttpMockFacade {
+  private _fetch: fetchMock.FetchMockSandbox;
+
+  constructor() {
+    this._fetch = fetchMock.sandbox();
+  }
+
+  // returns the locally mocked fetch instance
+  getFetch(): typeof node_fetch {
+    /// @ts-ignore
+    return this._fetch as typeof node_fetch;
```

Comment:
I had to ignore a type error here otherwise, the fetch-mock type is not compatible with node-fetch.